installation/index.md

```diff
@@ -49,11 +49,13 @@ OpenJDK may also be used, but it has some [known limitations](https://community.
 | [Oracle Java](https://java.com/en/)         | Full openHAB support on all platforms                                                                                                                                                 | [Licensing restrictions](https://blog.takipi.com/running-java-on-docker-youre-breaking-the-law/), manual installation required on many Linux systems: [Ubuntu](https://help.ubuntu.com/community/Java), [Mint](https://community.linuxmint.com/tutorial/view/1091), and [Debian](https://wiki.debian.org/Java/Sun) are some examples |
 | [OpenJDK](http://openjdk.java.net)          | Supported by many Linux distributions, packages [typically available](http://openjdk.java.net/install/index.html)                                                                     | [Performance issues](https://github.com/openhab/openhab-distro/issues/10#issuecomment-223786506) on ARM platforms, [compatibility issues](https://community.openhab.org/t/openhab-is-offline-message-fixed/17441/8) with certain bindings and certificates                                                                           |
 
-Please download and install the Java 8 version of the JVM. 
-openHAB is known not to work on Java 9 and Java 10.
+Please download and install the **Java 8** version of the JVM.
+openHAB does not work well with newer Java versions such as Java 9 or Java 11.
```

Comment:
See https://github.com/openhab/openhab-distro/issues/768. It does work. But it might not work well for everyone. :wink: